src/Microsoft.AspNet.Mvc.Core/ActionResults/ObjectResult.cs

```diff
@@ -40,9 +40,17 @@ public ObjectResult(object value)
 
         public override async Task ExecuteResultAsync(ActionContext context)
         {
+            var logger = context.HttpContext.RequestServices.GetRequiredService<ILoggerFactory>()
+                .CreateLogger<ObjectResult>();
+                            
             // See if the list of content types added to this object result is valid.
             ThrowIfUnsupportedContentType();
             var formatters = GetDefaultFormatters(context);
+
+            logger.LogVerbose(
+                "Available output formatters for writing the response: {OutputFormatters}",
+                formatters.Select(f => f.GetType().FullName));
```

Comment:
I wouldn't include this, it seems like overkill. This is either coming from configuration and is the same for everything, or it's something the user put in their code.